doc/man3/X509_load_http.pod

```diff
@@ -0,0 +1,50 @@
+=pod
+
+=head1 NAME
+
+X509_load_http,
+X509_http_nbio,
+X509_CRL_load_http,
+X509_CRL_http_nbio
+- certificate and CRL loading functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/x509.h>
+
+ X509 *X509_load_http(const char *url, int timeout);
+ X509_CRL *X509_CRL_load_http(const char *url, int timeout);
+
+ #define X509_http_nbio(url)     X509_load_http(url, 0)
+ #define X509_CRL_http_nbio(url) X509_CRL_load_http(url, 0)
```

Comment:
As above - should there be an option to provide the underlying BIO?

Reply:
Yes.
Analogously to above, the`url` parameter is still used/needed, e.g., to fill in the HTTP header.